CHANGELOG-prerelease.md

```diff
@@ -3,6 +3,7 @@
 - [](https://github.com/patternfly/patternfly-elements/commit/) fix: pfe-tabs adding dynamic tab in IE11 #838
 - [eb74cb8](https://github.com/patternfly/patternfly-elements/commit/eb74cb8f989048164fbb6ed1508c502659a752ed) feat: add event emission to pfe-select
 - [](https://github.com/patternfly/patternfly-elements/commit/) fix: Add a warning about updating the on attribute before upgrade
```

Comment:
@starryeyez024 Generally it's helpful to add the hash link for the item(s) in the log before yours (and the person committing next will do that for your addition too).  Less work during the release.